tests/modules/fgbio/groupreadsbyumi/main.nf

```diff
@@ -3,14 +3,14 @@
 nextflow.enable.dsl = 2
 
 include { FGBIO_GROUPREADSBYUMI } from '../../../../modules/fgbio/groupreadsbyumi/main.nf'
+params.groupreadsbyumi_strategy = 'Adjacency'
 
 workflow test_fgbio_groupreadsbyumi {
 
     input = [
         [ id:'test', single_end:false ], // meta map
         file(params.test_data['homo_sapiens']['illumina']['test_paired_end_umi_unsorted_tagged_bam'], checkIfExists: true)
     ]
-    strategy = "Adjacency"
 
-    FGBIO_GROUPREADSBYUMI ( input, strategy )
+    FGBIO_GROUPREADSBYUMI ( input, params.groupreadsbyumi_strategy )
```

Comment:
```suggestion
    FGBIO_GROUPREADSBYUMI ( input, 'Adjacency' )
```

Reply:
I think we can simplify the test this way

Reply:
definitely!